run.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
I would change name of this file to something like `prepare_data.sh` or `prepare_massive_dataset.sh` or `get_data.sh` or `get_massive_dataset.sh` etc. because `run.sh` is too vague.